dev/io.openliberty.wsoc.internal_fat/bnd.bnd

```diff
@@ -0,0 +1,45 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src:\
+    fat/src,\
+    test-applications/basic.jar/src,\
+    test-applications/basic.war/src,\
+	test-applications/websocket11.war/src
+
```

Comment:
Can you just clean up this white space difference ?

Reply:
Sounds good, Ill make the change